x-pack/plugins/maps/public/util.ts

```diff
@@ -156,3 +158,26 @@ export async function fetchGeoJson(
     })
   );
 }
+
+export const addFeatureToIndex = async (
```

Comment:
Nit: since these methods are so coupled with elasticsearch, I would recommend moving them closer to where they are used and putting them in "x-pack/plugins/maps/public/classes/sources/es_search_source" folder